Scarb.lock

```diff
@@ -3,12 +3,18 @@ version = 1
 
 [[package]]
 name = "cubit"
-version = "1.2.0"
-source = "git+https://github.com/influenceth/cubit.git#b4590530d5aeae9aabd36740cc2a3d9e6adc5fde"
+version = "1.3.0"
+source = "git+https://github.com/akhercha/cubit.git?branch=chore%2Fcairo_upgrade#d3869a3f0c47e5ed229bbbfe2fce3fc0510cbc8a"
```

Comment:
You should keep the cubit version as is – we're still using Cairo 2.3.1 as 2.4.0 is not deployable on Starknet